### PR TITLE
Add Excalibur article

### LINKED ISSUE #4858: Excalibur [5/31/2021]
* [Draft](https://docs.google.com/presentation/d/1Coe7Fv1hIP2sHrqRHQe5CnsmrtHiIUifu4An8hNi71k/edit?usp=sharing)
* [Discussion](https://groups.google.com/a/google.com/g/chrome-devrel-content/c/_Hcwymm8aJg/m/2hag9y2TAQAJ)

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #4858

Changes proposed in this pull request:

- Add article

### CODE REVIEW

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:68]:
```suggestion
I click the save icon and enter a file name in the file save dialog. In Chrome, a browser that
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:179]:
```suggestion
Now to saving. In Excalidraw, saving happens in a function called `saveAsJSON()`. It first serializes
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:181]:
```suggestion
`fileSave()`. This function is likewise provided by the browser-fs-access library.
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:256]:
```suggestion
support the File System Access API, I can then even immediately save my changes. No need to go
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:304]:
```suggestion
so when you double-clicked an `.excalidraw` file, the Excalidraw Electron app would open. Lipis, whom
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:321]:
```suggestion
Handling. 

This is a regular macOS Big Sur installation. Now check out what happens when I
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:323]: >it's just less dramatic to demonstrate in a screencast.

Stuff like this makes me think you should included some screen shots in this article. I don't want to hold up this PR though. If you do, we can merge it later.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:347]:
```suggestion
the operating system. I do this in a new field called `file_handlers` in the Web App Manifest. Its
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:348]:
```suggestion
value is an array of objects with an action and an `accept` property. The action determines the URL
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:369]:
```suggestion
The next step is to handle the file when the application launches. This happens in the `launchQueue`
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:370]:
```suggestion
interface where I need to set a consumer by calling, well, `setConsumer()`. The parameter to this
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:371]:
```suggestion
function is an asynchronous function that receives the `launchParams`. This `launchParams` object has a
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:373]:
```suggestion
and from this file handle I get a blob that I then pass to our old friend `loadFromBlob()`.
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:408]:
```suggestion
onto the clipboard by passing a one-element array with a `ClipboardItem` with the blob to the
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:409]:
```suggestion
`navigator.clipboard.write()` function. For more information on what you can do with the clipboard API,
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:419]: I'm thinking about images again.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:421]: Or even a 5 second autoplay video.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:437]: I realize this is a transcript, but the English idiom is "speaking of which".

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:442]: This has made it into Websters and Oxford as two words, which means we would need change Google properties to match if one word is more common.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:451]:
```suggestion
early version of tabbed application mode in Excalidraw. This is how it looks.
```

Comment (jpmedley) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:451]: This is one place where we must have an image before merging.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:464]:
```suggestion
have shown today pop up in your apps. My name is Tom, you can find me as
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Because this is a transcript, I stayed away from changing wording. There are a couple of places where I felt I had to add or replace a word. My reasons are explained.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (tomayac) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:83]: Fill is fine here. "I end up with multiple files […] that fill up my downloads folder".

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (tomayac) [inline src/site/content/en/blog/excalidraw-and-fugu/index.md:153]:
```suggestion
The fallback implementation relies on an input element of type "file". After the negotiation
of the to-be-accepted mime types and extensions,
the next step is to programmatically click the input element so
```

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (tomayac) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:419]: Images are still coming.

File: src/site/content/en/blog/excalidraw-and-fugu/index.md
Comment (jpmedley) [reply src/site/content/en/blog/excalidraw-and-fugu/index.md:445]: FYI, dictionaries don't dictate. They document what's done.

**Comment by jpmedley** [comment]
I can't approve this while it's in draft state.

**Comment by tomayac** [comment]
@jpmedley All images (missing `alt` texts are a known issue, will fix tomorrow) and screencasts are done now. This is ready for full review.

**Comment by jpmedley** [comment]
@tomayac I had considered what I already did a full review. I'm ok with whatever images you select. Merge as you see fit.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**
